### ISSUE TITLE
[icetray] `name` really not needed as a required parameter in `icetray.traysegment` (Trac #1635)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1627]
With `tray.Add` not needing a user-defined name, `icetray.traysegment` really doesn't need it as an necessary input parameter anymore 

Migrated from https://code.icecube.wisc.edu/ticket/1635
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T20:45:37", 
    "description": "With `tray.Add` not needing a user-defined name, `icetray.traysegment` really doesn't need it as an necessary input parameter anymore ", 
    "reporter": "rmaunu", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547239537194558", 
    "component": "combo core", 
    "summary": "[icetray] `name` really not needed as a required parameter in `icetray.traysegment`", 
    "priority": "normal", 
    "keywords": "icetray, traysegment", 
    "time": "2016-04-17T03:39:54", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160417T16:38:40: 
* olivas changed owner from "" to "olivas"
* olivas commented:

> In changeset 2518:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2518"
> segments don't need a name. closes #1627.
> ```


* olivas changed status from "new" to "closed"
* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:45:37: nwhitehorn commented:

> In changeset 168725:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168725"
> segments don't need a name. closes #1627.
> ```
